mne/preprocessing/ica.py

```diff
@@ -195,9 +195,10 @@ class ICA(ContainsMixin):
         Additional parameters passed to the ICA estimator as specified by
         ``method``.
     max_iter : int
-        Maximum number of iterations during fit. Defaults to 200. The actual
-        number of iterations it took :meth:`ICA.fit` to complete will be stored
-        in the ``n_iter_`` attribute.
+        Maximum number of iterations during fit. Defaults to 1000 for ``'fastica'``
```

Comment:
```suggestion
        Maximum number of iterations during fit. If None, it will default to 1000 for ``'fastica'``
```

Reply:
@agramfort, @hoechenberger - Thank you for the suggestions! As you can see above, Richard and other mentors discussed whether to make the default 'auto', and I already implemented that locally (just haven't pushed the changes yet); do you think the None option would be better?

Reply:
+1 for 'auto' and disabling None option.